hedera-node/src/main/java/com/hedera/services/state/expiry/ExpiryRecordsHelper.java

```diff
@@ -108,19 +111,29 @@ public void streamCryptoRenewal(
                         .setHbarAdjustments(sideEffectsTracker.getNetTrackedHbarChanges())
                         .setStakingRewardsPaid(sideEffectsTracker.getStakingRewardsPaid())
                         .setFee(fee);
-        stream(expirableTxnRecord, synthBody, eventTime);
+        finalizeAndStream(expirableTxnRecord, synthBody, eventTime, recordStreaming);
     }
 
-    private void stream(
+    public static void finalizeAndStream(
```

Comment:
Probably we can add `StreamingUtils` for these static methods ?

Reply:
💯 Will add in touch-up PR.